src/langs/fr/messages.json

```
@@ -66,5 +71,26 @@
   "IMPORT_ZIP": "L'archive ZIP a été importé avec succès",
   "IMPORT_H5P": "La librairie H5P a été importé avec succès",
   "DELETE_MEMBER": "Le compte a été supprimé avec succès",
-  "UPDATE_PASSWORD": "Le mot de passe a été mis à jour avec succès"
+  "UPDATE_PASSWORD": "Le mot de passe a été mis à jour avec succès",
+
+  "INVALID_ARCHIVE_FILE": "Le fichier soumis n'est pas une archive zip",
+  "INVALID_FILE_ITEM": "Les propriétés du fichier sont invalides",
+  "UNEXPECTED_EXPORT_ERROR": "Une erreur est survenue durant l'exportation de l'archive zip",
+
+  "NOT_APP_ITEM": "L'élément cible n'est pas une application",
+  "INVALID_APP_ORIGIN": "L'application ne correspond pas à l'origine enregistrée",
+  "TOKEN_ITEM_ID_MISMATCH": "Le jeton d'authentification ne correspond pas à l'élément cible",
+  "APP_DATA_NOT_FOUND": "Aucune données d'application trouvés",
```

```suggestion
  "APP_DATA_NOT_FOUND": "Aucune donnée d'application trouvée",
```
or
```suggestion
  "APP_DATA_NOT_FOUND": "Aucunes données d'application trouvées",
```
though the latter is not technically correct (https://www.larousse.fr/dictionnaires/francais/aucun/6369)